observability/pipeline.yaml

```diff
@@ -0,0 +1,36 @@
+$schema: "pipeline.schema.v1"
+serviceGroup: Microsoft.Azure.ARO.HCP.Observability
+rolloutName: Observability Rollout
+resourceGroups:
+- name: '{{ .global.rg }}'
+  subscription: '{{ .global.subscription }}'
+  steps:
+  - name: mirror-image
+    action: Shell
+    command: ../image-sync/on-demand/sync.sh
+    dryRun:
+      variables:
+      - name: DRY_RUN
+        value: "true"
+    variables:
+    - name: TARGET_ACR
+      configRef: acr.svc.name
+    - name: SOURCE_REGISTRY
+    - name: PULL_SECRET_KV
+      configRef: global.keyVault.name
+    - name: PULL_SECRET
+      configRef: imageSync.componentSync.pullSecretName
+- name: '{{ .svc.rg  }}'
+  subscription: '{{ .svc.subscription  }}'
+  aksCluster: '{{ .svc.aks.name  }}'
+  steps:
+  - name: deploy
+    action: Shell
+    command: make deploy
+    dryRun:
+      variables:
+      - name: DRY_RUN
+        value: "true"
+    variables:
+    - name: DEPLOY
+      configRef: svc.jaeger.deploy
```

Comment:
Will we need another service group to do this individually on the svc and mgmt cluster?

Reply:
But perhaps a setting for svc and mgmt is also sufficient? wdyt  @simonpasquier @dinhxuanvu ?

Reply:
It is :) 
I just moved the file into `observability/tracing`.